crimsobot/cogs/image.py

```diff
@@ -29,6 +31,39 @@ class Image(commands.Cog):
     def __init__(self, bot: CrimsoBOT):
         self.bot = bot
 
+    async def get_previous_image(self, ctx: commands.Context) -> str:
```

Comment:
this probably doesn't need to totally be here (maybe imagetools or something) but i'm fine with it.

Reply:
I debated this...bah. I'll leave it for now.